src/options-schema.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
`src/options-schema.json` => `./options.json`, if  it should reside in the src and needs to be copied, the change needs to be made in  `webpack-defaults` beforehand as always 😛